src/main/java/net/sf/mzmine/modules/visualization/spectra/PlotMode.java

```diff
@@ -0,0 +1,24 @@
+/*
```

Comment:
That is correct.  It is not there and is needed for one of the
visualizations that PI added as it contains the defines that are not the
same as what MZmine uses.  I mentioned this in an email to you.

David

On Fri, Oct 9, 2015 at 9:59 PM, TDI-tenderholt <notifications@github.com>
wrote:

> In src/main/java/net/sf/mzmine/modules/visualization/spectra/PlotMode.java
> <https://github.com/TDI-tenderholt/mzmine2/pull/1#discussion_r41666117>:
>
> > @@ -0,0 +1,24 @@
> > +/*
>
> This file doesn't appear to be in the upstream project.
> https://github.com/mzmine/mzmine2/tree/master/src/main/java/net/sf/mzmine/modules/visualization/spectra
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/TDI-tenderholt/mzmine2/pull/1/files#r41666117>.
>